paasta_tools/api/api_docs/swagger.json

```diff
@@ -1113,9 +1113,25 @@
                     "format": "int32",
                     "description": "The number of desired instances of the service"
                 },
+                "create_timestamp": {
+                    "type": "number",
+                    "format": "float",
+                    "description": "Unix timestamp when this app was created"
+                },
+                "namespace": {
+                    "type": "string",
+                    "description": "The namespace this app is running"
+                },
+                "pods": {
+                    "type": "array",
+                    "description": "Pods associated to this app",
+                    "items": {
+                        "$ref": "#/definitions/KubernetesPod"
+                    }
+                },
                 "error_message": {
                     "type": "string",
-                    "description": "Error message when a marathon job ID cannot be found"
+                    "description": "Error message when a kubernetes app ID cannot be found"
```

Comment:
app ID?

Reply:
I just follow the name of the property 'app_id' at line 1075. Do you have better name?

Reply:
What about "kubernetes object" ?